p2p/node.go

```diff
@@ -63,12 +63,6 @@ type QriNode struct {
 	// localResolver allows the node to resolve local dataset references
 	localResolver dsref.Resolver
 
-	// handlers maps this nodes registered handlers. This works in a way
-	// similary to a router in traditional client/server models, but messages
-	// are flying around all over the place instead of a
-	// request/response pattern
-	handlers map[MsgType]HandlerFunc
```

Comment:
This is only used for the `depQriProtocolID` protocol. All stream handling for the `depQriProtocolID` has been moved to func `depQriStreamHandler` in `p2p_deprecate.go`